src/main/java/core/basesyntax/bookstore/dto/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.bookstore.dto;
+
+import core.basesyntax.bookstore.validation.FieldMatch;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
+import lombok.Data;
+
+@Data
+@FieldMatch(first = "password", second = "repeatPassword", message = "Passwords do not match")
+public class UserRegistrationRequestDto {
+    @NotBlank
+    @Email
+    private String email;
+    @NotBlank
+    @Size(min = 8, max = 20)
+    private String password;
+    @NotBlank
+    @Size(min = 8, max = 20)

```

Comment:
[read about Size and Length ](https://www.baeldung.com/jpa-size-length-column-differences)annotations and choose one to use in the project